nautobot/core/utils/__init__.py

```diff
@@ -306,6 +306,9 @@ def to_meters(length, unit):
     """
     Convert the given length to meters.
     """
+    # Avoid circular import
+    from nautobot.dcim import choices
```

Comment:
Probably out of scope for this PR, but wonder if we should change `dcim.choices.CableLengthUnitChoices` to something more generic like `core.choices.LengthUnitChoices` so as to not need this indirect import?